test/test_examples_3d.jl

```diff
@@ -22,6 +22,67 @@ const EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "
+    test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_advection_basic.jl"),
+      l2   = [0.002727292086517533],
+      linf = [0.024833049753677727],
+      initial_condition=Trixi.initial_condition_sin)
+  end
+
+  @testset "taal-confirmed elixir_advection_basic.jl with initial_condition_constant" begin
+    test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_advection_basic.jl"),
+      l2   = [9.770171014620371e-16],
+      linf = [2.4424906541753444e-15],
+      initial_condition=Trixi.initial_condition_constant)
+  end
+
+  @testset "taal-confirmed elixir_advection_basic.jl with initial_condition_linear_z and periodicity=false" begin
+    test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_advection_basic.jl"),
+      l2   = [6.607840408143593e-16],
+      linf = [5.773159728050814e-15],
+      initial_condition=Trixi.initial_condition_linear_z,
+      boundary_conditions=Trixi.boundary_condition_linear_z, periodicity=false)
+  end
+
+  @testset "taal-confirmed elixir_euler_source_terms.jl" begin
+    test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_source_terms.jl"),
+      l2   = [0.010323099666828388, 0.00972876713766357, 0.00972876713766343, 0.009728767137663324, 0.015080409341036285],
+      linf = [0.034894880154510144, 0.03383545920056008, 0.033835459200560525, 0.03383545920054587, 0.06785780622711979])
+  end
+
+  @testset "taal-confirmed elixir_euler_source_terms.jl with split_form" begin
+    test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_source_terms.jl"),
+      l2   = [0.010323099666828388, 0.00972876713766357, 0.00972876713766343, 0.009728767137663324, 0.015080409341036285],
+      linf = [0.034894880154510144, 0.03383545920056008, 0.033835459200560525, 0.03383545920054587, 0.06785780622711979],
+      volume_integral=VolumeIntegralFluxDifferencing(flux_central))
+  end
+
+  @testset "taal-confirmed elixir_euler_eoc_test.jl" begin
+    test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_euler_eoc_test.jl"),
+      l2   = [0.000363723832448333, 0.00039555684672049366, 0.0003955568467203738, 0.00039555684672064724, 0.0007811604790242773],
+      linf = [0.002400072140187337, 0.0029635489437536133, 0.0029635489437540574, 0.0029635489437565, 0.007191455734479657])
+  end
+
+  @testset "taal-check-me elixir_eulergravity_eoc_test.jl" begin
+    test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_eulergravity_eoc_test.jl"),
+      l2   = [0.00042767201750631214, 0.0004720121013484361, 0.00047201210134851195, 0.00047201210134847107, 0.0010986046486879376],
+      linf = [0.003497353351708421, 0.0037653614087260756, 0.003765361408728074, 0.0037653614087242993, 0.008372792646797134],
+      resid_tol = 1.0e-4, tspan = (0.0, 0.2))
+  end
+
+  @testset "taal-confirmed elixir_advection_mortar.jl" begin
+    test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_advection_mortar.jl"),
+      l2   = [0.0018461483161353273],
+      linf = [0.017728496545256434])
+  end
```

Comment:
Could you please sort the tests by equation?

Reply:
I deliberately tried to reproduce the non-sorting of the Taam tests. This helps (me) to keep track of corresponding tests between Taal and Taam. My idea was to do a final sorting after all tests have been converted & confirmed, i.e., when we do not care about Taam/Taal comparisons anymore. I was also wondering if we should split the tests (at least for 2D and 3D) by equation, such that we can further parallelize testing on GH Actions and if someone wants to test specific changes.

I can either create an issue for this or amend one of the TODO lists for Taal (I am not sure which one is the right one - the one in the description of #200?)

Reply:
I think it's fine to open an issue and mention that as blocking at the bottom of the opening post in #200.

Reply:
Then, we can also discuss whether we really need tests on 3 OS for every PR. Maybe I was a bit too greedy and it's enough to just use one OS. In the end, we're just using pure Julia in Trixi (and the non-Julia parts such as MPI should be tested on different OS in their own repos).

Reply:
> I think it's fine to open an issue and mention that as blocking at the bottom of the opening post in #200.

Done (#301) and done.